### ISSUE TITLE
10. binder

### ISSUE THREAD

**Issue by rkdarst** [AaltoSciComp/python-for-scicomp#12]
Check the pad, but as of this writing:

Binder:
- Binderize a repository
- Put on Zenodo and get a DOI

**Comment by annefou** [comment]
Exercise:
https://coderefinery.github.io/jupyter/06-sharing/#exercise-making-your-notebooks-reproducible-by-anyone-via-binder

Optional exercise:
https://coderefinery.github.io/jupyter/06-sharing/#optional-exercise-what-happens-without-requirementstxt

Going further: JupyterBook
https://jupyterbook.org/intro.html

**Comment by rkdarst** [comment]
I think it would be a fine idea to include only links to the external
resources (and improve those), instead of re-writing it ourselves.
What do you think?

(general idea for all episodes)

**Comment by bast** [comment]
> I think it would be a fine idea to include only links to the external resources (and improve those), instead of re-writing it ourselves. What do you think? (general idea for all episodes)

Yes, whenever possible, I plan to improve and link instead of copy and improve.

**Comment by annefou** [comment]
Good point. Yes it is probably a good idea to avoid copy-paste (because here we manage both material).

**Comment by annefou** [comment]
Shall I close this issue? (Sorry I forgot to reference it in my PR...
